data_validation/__main__.py

```diff
@@ -479,9 +480,9 @@ def main():
     elif args.command == "configs":
         run_validation_configs(args)
     elif args.command == "find-tables":
-        print(find_tables_using_string_matching(args))
+        logging.info(find_tables_using_string_matching(args))
     elif args.command == "query":
```

Comment:
We could standardize the logging for standardize and easier debugging
for instance the above could be changed to 

logging.info(f"LInfo: {find_tables_using_string_matching(args)}")

Similarly we could use -
LWarn for Warnings
LExcept for Exceptions and so on